packages/-ember-data/tests/integration/adapter/rest-adapter-test.js

```diff
@@ -952,7 +958,13 @@ module('integration/adapter/rest_adapter - REST Adapter', function (hooks) {
   });
 
   test('findMany - findMany uses a correct URL to access the records', async function (assert) {
-    Post.reopen({ comments: DS.hasMany('comment', { async: true }) });
+    Post = DS.Model.extend({
```

Comment:
same as above, declare `Post` here instead of assigning to the variable with the greater scope access above. I'd do a sweep since there are likely more instances of this. If you can delete the `Post` variable declaration on line 23 you'll know you were successful.